### ISSUE TITLE
"location" appears in string files

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/energy-skate-park#171]
Over in https://github.com/phetsims/phet-info/issues/126, we are changing "location" to "position throughout the code base.   I noted that energy-skate-park has occurrences of "location" in *strings*.json files.   

In https://github.com/phetsims/phet-info/issues/126#issuecomment-568008965, @jbphet said:
> I was going to respond to @pixelzoom and request that string keys not be changed, but I looked at this a bit and all occurrences of "location" seem to be in translations with no corresponding string in the English file.  I'm not quite sure what's going on here.  There's a commit - https://github.com/phetsims/energy-skate-park-basics/commit/3e689653b2a2b2299d0ec933ff3427a593079040 - where the strings with "location" in their keys were removed from the strings file, but how did they end up getting translated?  Rosetta is specifically designed to only present to the user strings that are used in the sim.
> 
> Perhaps the bottom line is this - if @samreid or @jessegreenberg can confirm that these strings are not used in the currently live version of the sim, nor in the master version, the string keys in question in the translated string files can be changed, left alone, or removed from the translations files, since they will have no effect on the translated sim.

Assigning to @samreid and @jessegreenberg to decide what (if anything) to do.

**Comment by samreid** [comment]
It looks like ESP and ESPB do not have any location strings in the english files.  I'll leave this up to @jessegreenberg to confirm or comment.

**Comment by jessegreenberg** [comment]
> - if @samreid or @jessegreenberg can confirm that these strings are not used in the currently live version of the sim, nor in the master version, the string keys in question in the translated string files can be changed, left alone, or removed from the translations files, since they will have no effect on the translated sim.

I verified that none of the strings removed in https://github.com/phetsims/energy-skate-park-basics/commit/3e689653b2a2b2299d0ec933ff3427a593079040 are used in live version (1.1) or in master. I am not sure how they got translated.

@jbphet do you think it is worth removing these unused translated strings? Is there a process for that or is it manual?

**Comment by jbphet** [comment]
It's a manual process - just edit the string files and commit them.  Just be sure they aren't used.  As far as whether to remove, I generally to do when I encounter a situation like this, but it doesn't hurt anything to keep them around.  So, I'd say it's up to you.

**Comment by jessegreenberg** [comment]
Thanks, that sounds good. I started to go through removing the unused strings but realized that this will take a substantial amount of time. https://github.com/phetsims/energy-skate-park-basics/commit/3e689653b2a2b2299d0ec933ff3427a593079040 removed 84 strings and each had been translated into 37 different languages. Removing these manually doesn't seem worthwhile at this point unless they are causing problems.

Closing this issue.
